docker/script/entrypoint.sh

```diff
@@ -81,14 +81,35 @@ fi
 
 case "$1" in
   local-runner)
+    # if S3_PLUGINS_PATH
+    if [ -n "$S3_PLUGINS_PATH" ]; then
+      echo "Downloading $S3_PLUGINS_PATH"
+      mkdir -p /usr/local/airflow/plugins
+      cd /usr/local/airflow/plugins
+      aws s3 cp $S3_PLUGINS_PATH plugins.zip
+      unzip -o plugins.zip 
+      rm plugins.zip
+    fi
+    # if S3_DAGS_PATH
+    if [ -n "$S3_DAGS_PATH" ]; then
+      echo "Syncing $S3_DAGS_PATH"   
+      mkdir -p /usr/local/airflow/dags
+      cd /usr/local/airflow/dags
+      aws s3 sync --exact-timestamp $S3_DAGS_PATH .
+    fi    
+    # if S3_REQUIREMENTS_PATH
+    if [ -n "$S3_REQUIREMENTS_PATH" ]; then
+      echo "Downloading $S3_REQUIREMENTS_PATH"
+      aws s3 cp $S3_REQUIREMENTS_PATH /usr/local/airflow/dags/requirements.txt
```

Comment:
If for some reason, `S3_DAGS_PATH` is missing, then the folder `/usr/local/airflow/dags` will not be present. 

Maybe there should be an error if `S3_DAGS_PATH` is not provided or we can place the requirements file in its own folder.